Makefile

```diff
@@ -316,10 +320,13 @@ docker-images-cassandra:
 	docker build -t $(DOCKER_NAMESPACE)/jaeger-cassandra-schema:${DOCKER_TAG} plugin/storage/cassandra/
 	@echo "Finished building jaeger-cassandra-schema =============="
 
+docker-images-elastic: TARGET = release
```

Comment:
I thought `--target` would be useful if a Dockerfile contained multiple build stages like `debug` and `release`. However, from what I see, there is only a `release` build stage in cmd/es-index-cleaner/Dockerfile. Would it be useful to still include the target param in this case?

Reply:
True, this can be removed. Done.